src/Dfc.CourseDirectory.WebV2/Features/NewApprenticeshipProvider/QANotifications.cshtml

```diff
@@ -1,53 +1,45 @@
 ﻿@using Microsoft.AspNetCore.Http.Extensions
-@using Microsoft.AspNetCore.Http;
-
 @model Dfc.CourseDirectory.WebV2.Features.NewApprenticeshipProvider.QANotificationsViewModel
 
-
 @if (Model.ProviderType.HasFlag(ProviderType.Apprenticeships))
 {
     @if (Model.Status.HasFlag(ApprenticeshipQAStatus.Submitted) || Model.Status.HasFlag(ApprenticeshipQAStatus.InProgress))
     {
         <pttcd-notification title="Quality assurance in progress" id="pttcd-new-apprenticeship-provider__qa-notifications-submitted">
-            <p>Once your quality assurance is complete you will be contacted with the outcome.</p>
+            <p class="govuk-body govuk-!-margin-bottom-0">Once your quality assurance is complete you will be contacted with the outcome.</p>
         </pttcd-notification>
     }
     else if (Model.Status.HasFlag(ApprenticeshipQAStatus.Failed))
     {
-        <pttcd-notification title="Quality assurance outcome" id="pttcd-new-apprenticeship-provider__qa-notifications-failed">
```

Comment:
Given that this component is already in use, presumably there's no issue with us shipping this updated copy for the old dashboard too?